pulsar/README.md

```diff
@@ -0,0 +1,86 @@
+# Pulsar Bindings
+This document defines how to describe Apache Pulsar specific information with AsyncAPI.
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+<a name="server"></a>
+## Server Binding Object
+
+This object contains information about the server representation in Pulsar.
+
+##### Fixed Fields
+
+Field Name | Type | Description | Applicability [default] |
```

Comment:
What about we have a table with the following headers instead:

* Field Name
* Type
* Required (yes/no)
* Description
* Default value (N/A when it's required)

Reply:
That works for me.
Glancing over the other bindings, there didn't appear to be a standard, but i kind of liked what the Kafka binding looked like

Reply:
Fixed. Have another look

Reply:
Yeah, bindings are still in diapers 😄 We'll slowly find a good pattern to follow.